### DISCUSSION TITLE
copilot suggests that selection sort is adaptive and stable which is Wrong!

### DISCUSSION THREAD

**Discussion by AKABharat** [github/copilot-docs#75] [Polls]
I recently came across a suggestion by Copilot in Visual Studio Code that selection sort is adaptive and stable. However, based on my understanding, selection sort is not typically considered adaptive and stable. I would like to initiate a discussion to gather insights and opinions from the community.

https://github.com/github/copilot-docs/assets/107737002/4cca91d4-53a0-4694-a0d7-9e3fb4a373c3

**Comment by allan1366** [comment]
来信收到，谢谢！张韶华

**Comment by AKABharat** [comment]
@allan1366  is there any update regarding the same above?
